abr_control/controllers/osc.py

```diff
@@ -21,6 +24,11 @@ class OSC(Controller):
         The max allowed velocity of the end-effector [meters/second].
         If the control signal specifies something above this
         value it is clipped, if set to None no clipping occurs
+    ctrlr_dof : 6D list of boolean, optional (Default: position control)
+        specifies which task space degrees of freedom are to be controlled
+        [x, y, z, alpha, beta, gamma] (Default: [x, y, z]
+        NOTE: if more ctrlr_dof are specified than degrees of freedom in
+        the robotic system, the controller will perform poorly
```

Comment:
should we add an `assert sum(ctrlr_dof) <= robot_config.N_JOINTS`? If not an assertion, maybe print a warning?

Reply:
hmm i wouldn't want to make it an assertion, but a warning is a good idea. i'll add this now